Base/Configurations/Debug.xcconfig

```diff
@@ -15,6 +15,9 @@ GCC_OPTIMIZATION_LEVEL = 0
 // Preproccessor definitions to apply to each file compiled
 GCC_PREPROCESSOR_DEFINITIONS = DEBUG=1
 
+// Allow testing
```

Comment:
👍 Just looked it up. It seems to be needed for "@testable" imports. Xcode complained on upgrading the project. Will add a line